sort.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+function bubbleSort($sort_array)
+{
+    $array_indexes = count($sort_array);
+
+    for ($i = 0; $i < $array_indexes; $i++) {
+        if (strpos($sort_array[$i], "\n") === false) {
+            $sort_array[$i] .= "\n";
+        }
+
+        for ($j = 0; $j < $array_indexes; $j++) {
```

Comment:
Are you sure $j's start and end values are correct?

Reply:
I.e. o and $array_indexes.

Reply:
yes absolutely, if you want me to use a different variant i can change it but the way it is works for me quite well